templates/al2023/provisioners/install-nodeadm.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+set -o nounset
+set -o errexit
+
+# use containerd to build nodeadm containerized via kaniko + nerdctl
+sudo systemctl enable --now containerd
+sudo nerdctl run \
+  --rm \
+  -v $PROJECT_DIR:/workspace \
+  gcr.io/kaniko-project/executor:latest \
```

Comment:
Is there a kaniko image hosted on ECR we can use?

Reply:
oh just saw this before commenting on the other thread; I don't see kaniko hosted on ECR and im not sure they have a reason to. is that a hard requirement on us hosting a mirror?

Reply:
we could make this a flag/configurable so that we can provide custom images for this if need be. The actual image can just be replicated to an ecr repo